examples/account/account_allowance_delete_transaction_nft.py

```
@@ -227,17 +257,22 @@ def verify_allowance_removed(spender_client, nft_id: NftId, owner_id: AccountId,
         )
 
         if receipt.status == ResponseCode.SPENDER_DOES_NOT_HAVE_ALLOWANCE:
-            print("Verification SUCCEEDED: Transfer failed with SPENDER_DOES_NOT_HAVE_ALLOWANCE as expected.")
+            print(
+                "Verification SUCCEEDED: Transfer failed with SPENDER_DOES_NOT_HAVE_ALLOWANCE as expected."
+            )
         elif receipt.status == ResponseCode.SUCCESS:
             print(f"Verification FAILED: Transfer succeeded unexpectedly!")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)